### DISCUSSION TITLE
site window size minimized with width, image error

### DISCUSSION THREAD

**Discussion by yeonsookKwak** [alshedivat/al-folio#2669] [Q&A] [answered]
git page build was clear and my problem is image broken during window resized.

when the web window size is full, everything is okay. However the width size changed, all images broken. 

Please help me.

**Comment by george-gca** [comment]
Can you point your repository and the page in your site where this happens?

**Comment by yeonsookKwak** [comment]
Hello, my repo is here https://yeonsookKwak/yeonsookkwak.github.io
And about.md page figure and projects make figure broken during zoom in window

**Reply by george-gca** [reply]
It is hard to say, there has been many changes since the version of the template you started your repo from. Did you make many changes? If not, I would recommend you starting over again from the latest version of our repo. Maybe this bug was already fixed in our code. You will also benefit from many features added to our code.

**Reply by yeonsookKwak** [reply, marked as answer]
Okay, I tried the latest version. And it works. Thanks for the quick feedback!
